<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatasetImporter</class>
 <widget class="QDialog" name="DatasetImporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Dataset</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="1" colspan="2">
      <widget class="QSpinBox" name="chip_y2"/>
     </item>
     <item row="3" column="0" colspan="4">
      <widget class="QLineEdit" name="featuresFile"/>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Alignment</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QSpinBox" name="chip_y1"/>
     </item>
     <item row="5" column="0">
      <widget class="QSpinBox" name="chip_x1"/>
     </item>
     <item row="12" column="0" colspan="4">
      <widget class="QLineEdit" name="mainImageFile"/>
     </item>
     <item row="14" column="0" colspan="4">
      <widget class="QLineEdit" name="secondImageFile"/>
     </item>
     <item row="6" column="0">
      <widget class="QSpinBox" name="chip_x2"/>
     </item>
     <item row="13" column="0" colspan="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Second image</string>
       </property>
      </widget>
     </item>
     <item row="14" column="4">
      <widget class="QToolButton" name="loadSecondImageFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="12" column="4">
      <widget class="QToolButton" name="loadMainImageFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Main image</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Features file</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QToolButton" name="loadFeaturesFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Dataset Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QLineEdit" name="datasetName"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Chip</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QSpinBox" name="alignment_a11">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QSpinBox" name="alignment_a12">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QSpinBox" name="alignment_a13">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QSpinBox" name="alignment_a23">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QSpinBox" name="alignment_a22">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QSpinBox" name="alignment_a21">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QSpinBox" name="alignment_a33">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QSpinBox" name="alignment_a32">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QSpinBox" name="alignment_a31">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DatasetImporter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DatasetImporter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
